### ISSUE TITLE
Keep alive not working after last update?

### ISSUE THREAD

**Issue by Aens** [benediktschmitt/py-ts3#71]
After the 3.2.0 update of teamspeak, my bots disconnect at 5minutes, seems like `keepalive` or `wait_for_event` is not working, know why? The error is:

```
File "D:/Proyectos/TS3Bot/v2/Ts3PyBotGw2.py", line 119, in log_in
    self.main_loop(nickname)
  File "D:/Proyectos/TS3Bot/v2/Ts3PyBotGw2.py", line 127, in main_loop
    event = ts3conn.wait_for_event(timeout=550)
  File "D:\Proyectos\TS3Bot\v2\ts3\query.py", line 391, in wait_for_event
    self._recv(timeout=remaining_time)
  File "D:\Proyectos\TS3Bot\v2\ts3\query.py", line 324, in _recv
    data = self._telnet_conn.read_until(b"\n\r", timeout=timeout)
  File "C:\Users\Aens\AppData\Local\Programs\Python\Python37-32\lib\telnetlib.py", line 327, in read_until
    return self.read_very_lazy()
  File "C:\Users\Aens\AppData\Local\Programs\Python\Python37-32\lib\telnetlib.py", line 403, in read_very_lazy
    raise EOFError('telnet connection closed')
EOFError: telnet connection closed
```

And the piece of code that raises that error is:

```
 def main_loop(self, nickname):
        """Create Main Loop to keep the program active"""
        while True:
            ts3conn.send_keepalive()
            try:
                # Send a message each X seconds or the bot will disconnect due to inactivity at 600 seconds
                event = ts3conn.wait_for_event(timeout=550)
           except ts3.query.TS3TimeoutError:
                print("{0} - ERROR: TIMED OUT".format(str(datetime.datetime.now())[:19]))
            else:
                self.event_handler(event, nickname)
```
```

**Comment by pathmann** [comment]
According to [this thread](https://forum.teamspeak.com/threads/136623-Query-idle-no-longer-working?highlight=invalid+command), the keepalive handling has changed. Instead of [sending a newline](https://github.com/benediktschmitt/py-ts3/blob/master/ts3/query.py#L394) you have to send a (valid or invalid) command.

**Comment by Aens** [comment]
I tried to change the keepalive command with next ones:  

```
self._telnet_conn.write(b"\n\r") # This is the current broken one
self._telnet_conn.write(b"keepmealive")
self._telnet_conn.write(b"keepmealive\n\r")
self._telnet_conn.write(b"whoami")
self._telnet_conn.write(b"whoami\n\r")
```

And... none of them work, same behaviour. I think I lack of the knowledge to understand where other fixes are needed.

**Comment by JF-Dunford** [comment]
I experienced the same issue as well. My resolution was to replace  `ts3conn.send_keepalive()` with `ts3conn.exec_"version"` at the start of the while loop in the script itself. Although, as pathmann referenced, you can use another valid or invalid command instead.

**Comment by pathmann** [comment]
[Server 3.3.1](https://forum.teamspeak.com/threads/136554-Release-TeamSpeak-3-Server-3-3-0-amp-3-3-1?p=460285#post460285) should accept the keepalive again (untested).
So only a few specific server versions are affected.

**Comment by JF-Dunford** [comment]
Unfortunately, the error still occurs with server version 3.3.1.
```
Traceback (most recent call last):
  File "P:/dev/examples/hello_bot.py", line 46, in <module>
    hello_bot(ts3conn)
  File "P:/dev/examples/hello_bot.py", line 31, in hello_bot
    event = ts3conn.wait_for_event(timeout=550)
  File "P:\dev\ts3\query.py", line 391, in wait_for_event
    self._recv(timeout=remaining_time)
  File "P:\dev\ts3\query.py", line 324, in _recv
    data = self._telnet_conn.read_until(b"\n\r", timeout=timeout)
  File "P:\dev\Python36\lib\telnetlib.py", line 327, in read_until
    return self.read_very_lazy()
  File "P:\dev\Python36\lib\telnetlib.py", line 403, in read_very_lazy
    raise EOFError('telnet connection closed')
EOFError: telnet connection closed
```

**Comment by Aens** [comment]
Are you sure, @JF-Dunford? I am trying and there is no issues anymore for me. It seems fixed in 3.3.1

**Comment by pathmann** [comment]
Keep in mind, that the default query timeout is 300 seconds ([Source](https://forum.teamspeak.com/threads/136622-Server-query-chat-close-after-a-while?p=459509#post459509)), so with timeout=550 you'll timeout with default server settings.
